pymc3/stats.py

```diff
@@ -581,8 +584,8 @@ def add_warns(*args):
         warnings.filterwarnings('always')
```

Comment:
This is catching a completely unrelated warning (DeprecationWarning('`logsumexp` is deprecated!\nImporting `logsumexp` from scipy.misc is deprecated in scipy 1.0.0. Use `scipy.special.logsumexp` instead.',)

One solution could be to replace the line `warnings.filterwarnings('always')` with the lines
```python
warnings.filterwarnings('always', 'For one or more samples')
warnings.filterwarnings('always', 'Estimated shape parameter')
```
This will specifically catch warnings with messages matching those strings. One problem with this solution is that if the messages inside `waic` and `loo` are changed `compare` will not longer be able to catch them (we should probably add a test...)

If you do not want to add this fix to the present PR, I could add it after merging this.